src/main/java/DCInvestigatorPlugin.java

```diff
@@ -185,14 +173,29 @@ private ErrorReport checkValidityofOutputVariable(OutputVariableImpl vv, PairedD
             //numCurves = numLifecyclesPerReal;
         }
         int prevLifecycles = real*numLifecyclesPerReal;
+
+        //Extracts realization number from dss record
+        String initialSim= outPdc.toString();
+        String[] simParse = initialSim.split("/",0);
+        String currentRealization = simParse[6].substring(2,8);
+        Integer curRealization = Integer.parseInt(currentRealization);
+
+        //Stores realization number in arraylist
+        ArrayList<Integer> realization = new ArrayList<Integer>();
+        realization.add(curRealization);
+
+        //get start lifecycle number
+        int startLifecycle = realization.get(0)*numLifecyclesPerReal;
```

Comment:
Is this variable "startLifecycle" different than "prevLifecycles"? See Line 175.